webapp/views.py

```diff
@@ -1071,6 +1072,27 @@ def thank_you():
     )
 
 
+def get_user_country_by_tz():
```

Comment:
Can you also write a comment here for this function explaining how it works? I guess the key of this system is to use `Intl.DateTimeFormat().resolvedOptions().timeZone;`